nilearn/regions/region_extractor.py

```diff
@@ -116,7 +116,7 @@ def _remove_small_regions(input_data, index, affine, min_size):
     # np.unique and then use np.bincount to count the region sizes.
 
     _, region_indices = np.unique(input_data, return_inverse=True)
-    region_sizes = np.bincount(region_indices)
+    region_sizes = np.bincount(region_indices.ravel())
```

Comment:
this seemed to fix the issue

the indices array were not one dimensional anymore

let's see what tests in CI say for the rest